azext_edge/edge/providers/support_bundle.py

```diff
@@ -86,6 +86,13 @@ def build_bundle(
         if ops_service in [OpsServiceType.auto.value, service_moniker]:
             deployed_apis = api_info["apis"].get_deployed()
 
+            if not deployed_apis:
+                expected_api_version = api_info["apis"].as_str()
+                logger.warning(
```

Comment:
this may impact the existing progress bar UX, have you tested this aspect?

Reply:
yeah I tested it, all the warnings are printed above the progress bar instead of in between, which looks good to me?
![image](https://github.com/user-attachments/assets/685536d3-bb01-4b6c-a39e-e28416725c26)

Reply:
ok cool